jib-core/src/main/java/com/google/cloud/tools/jib/api/JibContainerBuilder.java

```diff
@@ -239,6 +239,43 @@ public JibContainerBuilder addEnvironmentVariable(String name, String value) {
     return this;
   }
 
+  /**
+   * Sets the directories that the container will create as a volume. Volumes allows persistent
```

Comment:
Just some wording suggestions: I believe the volume definition in more in lines of "Sets the directories that may hold externally mounted volumes."

Reply:
Yep, that's is better. changed.